config/development.json

```diff
@@ -9,7 +9,8 @@
   "client_ID": "CIynn5wTPyYZQcA1FJx1Io9z4t7QWDaE",
   "features": {
     "autologin": "true",
-    "person_api_lookup": "true"
+    "person_api_lookup": "true",
+    "maintenanceMode": "true"
```

Comment:
nit: should use the same syntax for all variables
ie either blah_blah or blahBlah or blahblah :)